meta/meta.go

```diff
@@ -18,8 +18,8 @@ import (
 	infinispanv1 "github.com/infinispan/infinispan-operator/pkg/apis/infinispan/v1"
 	grafana "github.com/integr8ly/grafana-operator/v3/pkg/apis/integreatly/v1alpha1"
 	keycloakv1alpha1 "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
-	"github.com/kiegroup/kogito-cloud-operator/api/v1beta1"
-	kafkabetav1 "github.com/kiegroup/kogito-cloud-operator/core/infrastructure/kafka/v1beta1"
+	kafkabetav1 "github.com/kiegroup/kogito-operator/core/infrastructure/kafka/v1beta1"
+	v1 "github.com/kiegroup/rhpam-kogito-operator/api/v1"
```

Comment:
wouldn't better rename this import to rhpamv1 or something similar for better readability?

Reply:
done. renamed them to rhpamv1